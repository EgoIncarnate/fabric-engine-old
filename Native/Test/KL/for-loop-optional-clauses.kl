operator entry()
{
  Size i=0;
  for (;;)
  {
    report ++i;
    if ( i == 10 )
      break;
  }
}
