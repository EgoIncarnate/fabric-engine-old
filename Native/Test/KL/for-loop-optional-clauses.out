GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        StatementList
          AssignedVarDecl( VarDecl( 'i', Size ), ConstSize( 0 ) )
            ConstSize( 0 )
        CStyleLoop
          CompoundStatement
            StatementList
              Report
                UniOp( prefix ++ )
                  Var('i')
              ConditionalStatement
                BinOp( EQ )
                  Var('i')
                  ConstSize( 10 )
                BreakStatement
1
2
3
4
5
6
7
8
9
10
